assets/scripts/datalayer/endpoints.js

```diff
@@ -36,14 +37,19 @@
         purchase: {
           actionField: {
             id: orderId,
-            revenue: String($order.data('revenue')).replace(/,/g, ''),
-            tax: String($order.data('tax')).replace(/,/g, ''),
-            shipping: String($order.data('shipping')).replace(/,/g, ''),
+            revenue: String(order_data.revenue).replace(/,/g, ''),
```

Comment:
Why do we need to rename this?

Reply:
`orderData` is the array of data tags values retrieved from element `$order`.